### PR TITLE
add: `g4music`

### PR DESCRIPTION
**Pull request by soumyatheman**

### CODE REVIEW

**Comment by soumyatheman** [comment]
Why is the operation getting cancelled?

File: packages/g4music/g4music.pacscript [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+gives="g4music"
+version="1.8.1"
+build_depends="meson valac libgtk-4-dev libgstreamer1.0-dev libadwaita-1-dev"
+#This is a temporary fix, because pacstall doesn't install run-time deps before installation
+depends="libgtk-4-dev libgstreamer1.0-dev libadwaita-1-dev"
+url="https://gitlab.gnome.org/neithern/g4music/-/archive/v${version}/g4music-v${version}.tar.gz"
+repology=("project: g4music")
+description="Play your music in an elegant way"
+hash="3b7d2f47a08b6ce9a2bebf2ce0e4295aa07499628f18571483d809ca9270ab94"
+maintainer="সৌম্যদীপ <soumyadeepghosh2004@zohomail.in>"
```
Comment (Henryws) [inline packages/g4music/g4music.pacscript:11]:
```suggestion
maintainer="সৌম্যদীপ <soumyadeepghosh2004@zohomail.in>"

```

**Review by Henryws — CHANGES_REQUESTED** [review]
Put a newline before functions

File: packages/g4music/g4music.pacscript [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name="g4music"
+gives="g4music"
+version="1.8.1"
```
Comment (wizard-28) [inline packages/g4music/g4music.pacscript:3]:
```suggestion
version="1.8.2"
```

There's no `1.8.1` release, it was yanked probably.

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Update the `hash` as well

**Comment by Henryws** [comment]
@soumyatheman don't close and create new PRs.

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Run pre-commit hooks

**Comment by Henryws** [comment]
Please please please don't just use the default commit message for GitHub. Name them something meaningful.

**Comment by Henryws** [comment]
https://github.com/pacstall/pacstall-programs/blob/master/CONTRIBUTING.md

Please follow these from now on.

File: packages/g4music/g4music.pacscript [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+repology=("project: g4music")
+description="Play your music in an elegant way"
+hash="b4463737257f7985a48ce17a56b5a27a9a3ebfc02cf0f78b6725e03ca984dafe"
+maintainer="সৌম্যদীপ <soumyadeepghosh2004@zohomail.in>"
+
+build(){
+	meson setup build --buildtype=release
+}
+install(){
+	sudo meson install -C build --destdir=${pkgdir}
```
Comment (Henryws) [inline packages/g4music/g4music.pacscript:17]:
```suggestion
	sudo meson install -C build --destdir="${pkgdir}"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by soumyatheman** [comment]
My fork has become super messy due to me, I am reforking it and making a clean new PR again, so closing this

**Comment by Henryws** [comment]
Are you using git, or GitHub editor?

**Comment by soumyatheman** [comment]
I am using browser, to do all these, a big noob I am yet in git

**Comment by Henryws** [comment]
You must use git. You can ask for assistance in the general chat on Discord.

**Pull request closed without merging**